docs/src/charm/howto/upgrade-minor.md

```diff
@@ -0,0 +1,202 @@
+# How to Upgrade {{product}} to the next minor revision
+
+It is recommended that you keep your Kubernetes deployment
+updated to the latest available stable version. You should
+also update the other applications which make up Kubernetes.
```

Comment:
I would remove the second sentence and rephrase, or specify what other applications make up Kubernetes.

Reply:
thanks @eaudetcobello i'm opened to rewording this.  I guess i could list possible other charm integrations here `ceph-csi`, `cinder-csi`, `openstack-integrator` but there's like 60 charms and i'd rather not list them all.